### ISSUE TITLE
Add automatic flake detection into CI [Blocked: #59]

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#1760]
This is a bit more of a suggested solution, but I was thinking that we might be able to add automatic test flake detection for new PRs by:
1. Detecting new tests added in a PR (maybe just test suites to start)
2. Running these tests a certain # of times in parallel (which will be a lot easier to do with Bazel, so marking this as blocked on #59).

This will help protect against accidentally introducing newly flaky tests. It doesn't help against new code causing existing tests to become flaky (that will probably require detection against the develop branch rather than per PR branch).

**Comment by BenHenning** [comment]
Note that this might be more 'good third issue': this will be a bit hard for folks not familiar with Bazel.

**Comment by BenHenning** [comment]
In retrospect, I don't think this is going to be easily fixable so removing the starter label.

**Comment by BenHenning** [comment]
Duplicate of, and replaced by, #2641.
